src/Bridges/Nette/WebLoaderExtension.php

```diff
@@ -91,16 +91,12 @@ private function setupWebLoader()
 	{
 		$webLoader = $this->builder->addDefinition($this->prefix(self::ENGINE_PREFIX))
 			->setFactory(self::ENGINE_CLASSNAME)
-			->setArguments([$this->config['outputDir']]);
+			->setArguments([$this->config['outputDir'], $this->config['documentRoot']]);
```

Comment:
Thank you for the pull request!

I think that this change don't need to be a BC break. If the condition from below would be used here, then the parameter could still be optional. Could you please try it on your project where it cause problems? If it works, then the other changes could be reverted.

````PHP
$arguments = [$this->config['outputDir']];

if ($this->config['documentRoot']) {
    $arguments[] = $this->config['documentRoot'];
}

$webLoader = $this->builder->addDefinition($this->prefix(self::ENGINE_PREFIX))
    ->setFactory(self::ENGINE_CLASSNAME)
    ->setArguments($arguments);
````